hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/types/ClusterStateResponse.java

```diff
@@ -102,6 +102,12 @@ public final class ClusterStateResponse {
   @JsonProperty
   private long deletedDirs;
 
+  @JsonProperty
+  private String scmServiceId;
+
+  @JsonProperty
+  private String omServiceId;
```

Comment:
Nit: Let's add the `JsonProperty` filed (e.g. like "keysPendingDeletion") although `deletedDirs` does not have one.